### PR TITLE
Start of outline restructuring

### PR DESCRIPTION
**Pull request by noamross**
Didn't get quite as far as I wanted with this today but I did a fair bit of shifting around that I wanted you to have before you started in the morning! I think this is coming together.

### CODE REVIEW

**Comment by noamross** [comment]
(Sorry there's a fair bit of auto-reformatting.  **redoc** is not as seamless as I would hope in getting things back into the exact shape they were before).

File: outline.Rmd [hunk 1]
```diff
@@ -23,439 +21,671 @@ if (!file.exists ("statsoft.bib")) {
...
-It considering both of these issues of definition, it will be important to
-consider whether it may be advantageous or necessary to develop different
-procedures for different categories, whether in terms of categories of external
-form or categories of statistical software.
+statistical computing in all areas of empirical research.
 
+-   Supervised / regression models and algorithms
 
----
+-   Unsupervised models and algorithms
```
Comment (mpadge) [inline outline.Rmd:163]: Where are these list items from? They are not part of JSS scope, so you could please add a contextualizing sentence for them?

**Comment by mpadge** [comment]
Interesting to see what `redoc` did - mostly pretty impressive, assuming you did the round-trip conversion to and from a word doc? Only really the citations got truly mucked up; the rest survived largely intact. I knew about the package, but had kept away as firmly as i keep away from all microsoft products, so it's instructive for me to have this indirect interaction with your `redoc` workflow. Nice.

File: outline.Rmd (hunk 1 above)
Comment (noamross) [reply outline.Rmd:163]: I didn't get to it to finishing restructuring this section but will. I was thinking something like:

- The other axis of scope is what constitutes "statistical software," which could potentially encompass many things, including but not limited to,
- [bulleted list]
- JSS defines it this way [others define it some other way?]
- Defining this empirically is difficult - see text analysis appendix.

**Pull request merged**